packages/edit-site/src/components/sidebar-navigation-screen-templates/index.js

```diff
@@ -123,14 +124,31 @@ export default function SidebarNavigationScreenTemplates() {
 								</TemplateItem>
 							) ) }
 							{ ! isMobileViewport && (
-								<SidebarNavigationItem
-									className="edit-site-sidebar-navigation-screen-templates__see-all"
-									{ ...browseAllLink }
-									children={
-										config[ postType ].labels.manage
-									}
-									withChevron
-								/>
+								<>
+									<SidebarNavigationItem
+										className="edit-site-sidebar-navigation-screen-templates__see-all"
+										{ ...browseAllLink }
+										children={
+											config[ postType ].labels.manage
+										}
+										withChevron
+									/>
+									{ !! config[ postType ].labels
+										.reusableBlocks && (
+										<SidebarNavigationItem
+											href="edit.php?post_type=wp_block"
+											onClick={ () => {
```

Comment:
I think we'll have to use a link for accessibility purposes. I know that we probably will soon change this behavior to not require a link, but it seems like the fix is not that difficult either. I can push a commit to fix this :)!

Reply:
Done in [92319c8](https://github.com/WordPress/gutenberg/pull/50769/commits/92319c8d8c844b88d7772b09d012fa18daeae5c8).